spec/factories/repositories.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :repository do
```

Comment:
Same here, please make them all different and looking more like a real-life data